napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         url: str = '',
         summary: str = '',
         author: str = '',
-        license: str = "UNKNOWN",
+        license: str = "UNKNOWN",  # noqa: A002
```

Comment:
I'm confused — the flake8-builtins [readme](https://pypi.org/project/flake8-builtins/) says that A002 is "argument is shadowing a python builtin". But that's not happening here? Is there a way to annotate these noqa calls with what they mean, or does that make them invalid? e.g. `# noqa: A002 shadowing builtin`?

Reply:
... TIL that `license` is a Python built-in! 🤣